core/src/main/scala/scala/collection/parallel/ParIterableLike.scala

```diff
@@ -862,7 +862,7 @@ self =>
     def split = throw new UnsupportedOperationException("Does not split.")
   }
 
-  protected[this] trait NonDivisible[R] extends NonDivisibleTask[R, NonDivisible[R]]
+  protected[this] trait NonDivisible[R] extends NonDivisibleTask[R, NonDivisible[R]] { }
```

Comment:
I wonder if these extra braces solved the indentation issue. I solved it by putting the self-type in the traditional place.

Reply:
good call. fixed